### PR TITLE
Streamline comments and errors

### PR DESCRIPTION
**Pull request by chapati23**
### Description

Fixes https://github.com/mento-protocol/mento-general/issues/552

- [x] Made error messages across GoodDollar contracts more consistent
- [x] Made comments across GoodDollar comments more consistent
- [x] Introduced @inheritdoc to reduce comment duplication between interfaces and implementations in 0.8 contracts
- [x] Changed `Broker.configureTradingLimit()` from ~~public~~ to **external**

### CODE REVIEW

File: contracts/goodDollar/interfaces/IGoodProtocol.sol [hunk 1]
```diff
@@ -9,6 +9,7 @@ interface IGoodDollar {
 
   function safeTransferFrom(address from, address to, uint256 value) external;
 
+  // Only used in Fork Tests to give the Broker minting rights during testing
```
Comment (baroooo) [inline contracts/goodDollar/interfaces/IGoodProtocol.sol:12]: not sure if this comment makes sense since it is specific to how we use the interface

**Review by baroooo — APPROVED** [review]
LGTM ✅

**Review by bowd — APPROVED** [review]
❤️

File: contracts/goodDollar/interfaces/IGoodProtocol.sol (hunk 1 above)
Comment (chapati23) [inline contracts/goodDollar/interfaces/IGoodProtocol.sol:12]:
```suggestion
```

Comment (chapati23) [reply contracts/goodDollar/interfaces/IGoodProtocol.sol]: yeah, agreed, was just a stream-of-thought comment for me I think, removed

**Pull request merged**